### DISCUSSION TITLE
OnnxStableDiffusionXLPipeline.__init__() missing 3 required positional arguments: 'vae_decoder', 'text_encoder', and 'unet'

### DISCUSSION THREAD

**Discussion by eddy533** [lshqqytiger/stable-diffusion-webui-directml#415] [Q&A]
SO i get this when trying to run text2image with SD XL model with olive. standdard model worked fine. XL doesnt .  diffusers set to XL...anyone know?

**Comment by lshqqytiger** [comment]
This error occurs when the previous conversion/optimization failed. Find the corrupted cache folder in `models/ONNX/cache` and delete it.

**Reply by eddy533** [reply]
I did try removing it and re optimizing still gives  the same error. Also the only  diff pipline setting that works for me is onnx stable diffusion for SD1.5. Even I set to it "onnx stable diffusion upscale" and run text2image with highres fix upscale it throws 
"    TypeError: OnnxStableDiffusionUpscalePipeline.__call__() got an unexpected keyword argument 'height'"
I run on 7900xt win11

**Reply by lshqqytiger** [reply]
need full log
